__tests__/test-data.test.js

```diff
@@ -62,24 +63,23 @@ describe.only("GET /api/articles/:article_id", () => {
         expect(Array.isArray(arrayOfArticles)).toBe(true);
       });
   });
-  test.only("200,returns the correct obj ", () => {
+  test("200,returns the correct obj ", () => {
     return request(app)
       .get("/api/articles/2")
       .expect(200)
       .then((response) => {
         const arrayOfArticles = response.body.data;
         const articleId = arrayOfArticles[0].article_id;
         expect(articleId).toBe(2);
-        expect(arrayOfArticles[0]).toHaveProperty("article_id")
+        expect(arrayOfArticles[0].title).toBe(articleData[1].title);
+        expect(arrayOfArticles[0].topic).toBe(articleData[1].topic);
+        expect(arrayOfArticles[0].author).toBe(articleData[1].author);
+        expect(arrayOfArticles[0].body).toBe(articleData[1].body);
+        //        expect(arrayOfArticles[0]).toBe(articleData[2])
       });
   });
 
-  test("400, article must be a number ", () => {
-    return request(app)
-    .get("/api/articles/j")
-    .expect(404)
-  });
-  test("204, no content fouund  ", () => {
-    return request(app).get("/api/articles/500").expect(204);
+  test("400, no column found with htis specific name  ", () => {
```

Comment:
Should the test for 'no column found with this specific name' be for an 'invalid article_id'?

Also, is this a 400 or a 404 as you've expected 404 but put 400 in test description